nautobot/cloud/tables.py

```diff
@@ -39,6 +40,53 @@ class Meta(BaseTable.Meta):
         )
 
 
+class CloudNetworkTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    cloud_type = tables.Column(linkify=True)
+    cloud_account = tables.Column(linkify=True)
+    parent = tables.Column(linkify=True)
+    actions = ButtonsColumn(CloudNetwork)
+
+    class Meta(BaseTable.Meta):
+        model = CloudNetwork
+        fields = (
+            "pk",
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "prefixes",
```

Comment:
Have we checked if the prefixes column is rendered correctly?
I think ideally the column should render the number of prefixes with a link that will direct you to the filtered list of prefixes by this cloud network right?

Reply:
Yes, this column renders just fine, here is a screenshot. 
![image](https://github.com/nautobot/nautobot/assets/17092348/784ea4f3-7055-45da-ace2-fcf45e83df2d)

Reply:
Need to consider how this will work if there's a large number of prefixes, both from a performance and display standpoint.

Our *usual* convention is to use a `LinkedCountColumn` in such cases.

Reply:
Right `LinkCountColumn` is what I was describing, did not know that class existed.